packages/cli/src/concurrency/concurrency-control.service.ts

```diff
@@ -15,13 +16,15 @@ import { ConcurrencyQueue } from './concurrency-queue';
 export const CLOUD_TEMP_PRODUCTION_LIMIT = 999;
 export const CLOUD_TEMP_REPORTABLE_THRESHOLDS = [5, 10, 20, 50, 100, 200];
 
+export type ConcurrencyType = 'production' | 'evaluation';
```

Comment:
```suggestion
export type ConcurrencyQueueType = 'production' | 'evaluation';
```

Can we please also rename `type` to `queueType` elsewhere for clarity?